src/components/Routing.js

```diff
@@ -24,13 +28,27 @@ import { SignWithRouter } from './sign/Sign'
 
 import { handleRefreshAccount, handleRefreshUrl } from '../actions/account'
 
-import { ThemeProvider } from 'styled-components'
 import GlobalStyle from './GlobalStyle'
 const theme = {}
 
 const PATH_PREFIX = process.env.PUBLIC_URL
 
 class Routing extends Component {
+   constructor(props) {
```

Comment:
are we going to have language option button for wallet? We should test the translation is good enough or not. Since in our team, many language is known from different people. If need Chinese one, I will definitely help to see whether the translation is weird on the page.

Reply:
we probably going to start with just supporting `Accept-Language` https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.4